queries.sql

```diff
@@ -1,10 +1,59 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';  
+
+-- List the name of all animals born between 2016 and 2019.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                -- 
 SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
 SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
 SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+-- List name and escape attempts of animals that weigh more than 10.5kg
 SELECT name, escape_attempts FROM animals WHERE weight_kg  > 10.5;
+
+-- Find all animals that are neutered.
 SELECT * FROM animals WHERE neutered IS TRUE;
+
+-- Find all animals not named Gabumon.
 SELECT * FROM animals WHERE name <> 'Gabumon';
+
+-- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
```

Comment:
- [ ] Nice job on updating the database but please  you Inside a transaction:
* `UPDATE` the animals table by setting the species column to unspecified, then roll back the change

Reply:
Okay let me work on this.